### PR TITLE
impl Filecoin.MinerCreateBlock

### PR DESCRIPTION
**Pull request by LesnyRumcajs**
## Summary of changes

Changes introduced in this pull request:

- implemented `Filecoin.MinerCreateBlock`,
- changed wallet import so that it allows specifying the key via stdin non-interactively,
- both Lotus and Forest now share the same wallet key,
- first `write` tests - as a follow-up I'll uncomment ones that were blocked by this.

## Reference issue to close (if applicable)

Closes https://github.com/ChainSafe/forest/issues/4202

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

File: src/rpc_client/miner_ops.rs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+use crate::rpc::{
+    miner::{BlockMessage, BlockTemplate, MinerCreateBlock},
+    RpcMethod,
+};
+
+impl ApiInfo {
+    pub fn miner_create_block_req(block_template: BlockTemplate) -> RpcRequest<BlockMessage> {
+        RpcRequest::new(MinerCreateBlock::NAME, (block_template,))
+    }
+}
```
Comment (lemmih) [inline src/rpc_client/miner_ops.rs:14]: This looks like dead-code. `ApiInfo` is being replaced by `RpcMethod`.

**Review by lemmih — APPROVED** [review]

File: src/rpc_client/miner_ops.rs (hunk 1 above)
Comment (LesnyRumcajs) [reply src/rpc_client/miner_ops.rs]: Good catch!

**Review by aatifsyed — APPROVED** [review]
I'd love to see the `Miner` methods in `state.rs` moved while you're at it :)

**Comment by LesnyRumcajs** [comment]
> I'd love to see the `Miner` methods in `state.rs` moved while you're at it :)

Sure, I'll do it in a follow-up tomorrow.

**Pull request merged**